pkg/placement/raft/server.go

```diff
@@ -214,15 +214,15 @@ func (s *Server) StartRaft(ctx context.Context, sec security.Handler, config *ra
 		// Set default configuration for raft
 		s.config = &raft.Config{
 			ProtocolVersion:    raft.ProtocolVersionMax,
-			HeartbeatTimeout:   1000 * time.Millisecond,
-			ElectionTimeout:    1000 * time.Millisecond,
-			CommitTimeout:      50 * time.Millisecond,
+			HeartbeatTimeout:   2 * time.Second,
```

Comment:
Does increasing this time solve issue majorly or it's a temporary solution?

Reply:
I would love to understand _why_ this is hitting the timeout under load. Or there's always the risk that even with the timeout increased, it may be hit under heavier load.

Also, perhaps these should be configurable as CLI flags?

Reply:
This will be the new default going forward. I believe this should be configurable but we need to unblock users to deploy 1.12 to production. It is temporary in the sense that the ideal solution is to change how actor placement is discovered - but users cannot wait for that. We are fixing the plane while flying it.

Reply:
I want user to try an RC before we release this. I am working on a new test for Dapr but I need more time. We have a timing problem now.

Reply:
I suggest we make this a 1.12.4-rc.1 release first.

Reply:
I added a flag to pick the timeout settings.